index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/b21ce73c3c.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <div id="header_content">
+            <div id="logo">Progress Eze</div>
+            <nav>
+                <i class="fa-solid fa-bars"></i>
+            </nav>
+        </div>
+    </header>
+    
+    <main id="main">
+        <h1>Hey There. I’m Progress</h1>
+        <h2>I’m a software developer</h3>
+        <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        
+        <div class="social_handels">
+            <div class="social_icons">
+                <a  class="icons"href="https://github.com/Progress2002"><i class="fa-brands fa-github"></i></a>
+                <a class="icons" href="https://www.linkedin.com/in/progress-ezeamaka-27b114247"><i class="fa-brands fa-linkedin-in"></i></a>
+                <a  class="icons" href="https://twitter.com/Progress_2002">    <i class="fa-brands fa-twitter"></i>
+                </a>
+                <a class="icons mail-icon" href="ezeamakachisom2002@gmail.com">M</a>
+            </div>
+        </div>
```

Comment:
- [ ] I think it will be better if your lists of icons are wrapped in `<ul>` tag to group them since the icons are a list, and also wrap each icon in  a `li` tag with each icon being a link (`a` tag)

e.g. 

 ```
<ul class="social-media">
        <li id="angellist">
            <a href="">
                <i class="fa-brands fa-github"></i>
            </a>
        </li>
         <li id="medium">
            <a href="">
                <i class="fa-brands fa-twitter"></i>
            </a>
        </li>
       ...
    </ul>
```

- [ ]

Reply:
Modifications made